explainers/hidden-content-explainer.md

```diff
@@ -71,196 +75,152 @@ currently possible.
+performance of searching the content inside the `hidden=until-found` subtree
+because searching for text in these areas requires computing style and layout.
+`content-visibility` saves this computed style and layout instead of
+`display:none` which deletes them, which means that once the first run of
+find-in-page or ScrollToTextFragment will compute these values, and subsequent
+runs can reuse the saved style and layout.
 
 ## Accessibility
 
-`content-visibility: hidden-matchable` subtrees will not be included in the
-accessibility tree because they are not visible on the screen, just like
-`content-visibility: hidden`.
-
-The content inside `content-visibility: hidden-matchable` subtrees will still
-remain accessible to assisstive technology users because there should always be
-a way to reveal the content besides using find-in-page, such as a button to
-expand the content like a `<details>` element.
-
-## Scroll timing and element fragment issue
-
-With this feature, find-in-page and ScrollToTextFragment will have an
-asynchronous flow for firing the beforematch event and scrolling when the match
-is in a `content-visibility: hidden-matchable` subtree:
-1. A match within a `content-visibility: hidden-matchable` subtree has been
-   found
-2. Schedule an animation frame and wait until it happens
-3. Fire the beforematch event on the `hidden-matchable` ancestor before other
-   animation frame callbacks from script
-4. Schedule an animation frame and wait until it happens
-5. Scroll to the target match
-
-This behavior was implemented in response to early feedback about pages where
-the beforematch event handler queued an asynchronous task to remove the
-`hidden-matchable` property, in which case the requestAnimationFrame gap between
-firing the beforematch event and scrolling was needed in order to correctly
-scroll to the revealed content.
-
-Supporting beforematch for element fragments is problematic because scrolling
-for element fragments is always done synchronously. To be more specific,
-consider the following code:
-```html
-<div style="height:100px"></div>
-<div id=foo></div>
-<script>
-window.scrollY; // returns 0
-window.location.hash = '#foo';
-window.scrollY; // returns 100
-</script>
-```
-Since changes to the element fragment trigger synchronous scrolling which we
-can't change without breaking the current behavior, the beforematch event must
-be fired either synchronously before the scroll or via a posted task on the
-event loop instead of the rAF->beforematch->rAF->scroll behavior we have for
-ScrollToTextFragment and find-in-page.
-This means that pages which need a full rAF in between the beforematch event and
-the scroll in order to reveal the content to scroll to may not be able to reveal
-the target content in time for the scroll.
+`hidden=until-found` subtrees will not be included in the accessibility tree
+because they internally use `contenet-visibility:hidden`.
```

Comment:
I think we should be careful about stating that c-v is the reason for why they are not in the a11y tree. I think they aren't in the a11y tree because we don't want this content to be exposed to a11y since it goes against what the site author desires. The fact that we implement it using c-v hidden is tangential.

(also you have a typo in content-visibility)

Reply:
Ok, I merged it with some of the previous text. What do you think?

Reply:
That reads much better, thanks!